Website/index.html

```diff
@@ -42,7 +42,9 @@
           <a href="contact.html"><button class="button">Contact</button></a>
         </li>
         <li>
-          <div class="btn-toggle"><i id="symb" class="fa-solid fa-moon fb"></i></div>
+          <div class="btn-toggle"><i id="symb" class="fa-solid fa-moon fb"><div class="mode-tooltip">Dark/Bright Mode</div></i>
```

Comment:
@Nitesh232 Please change **Bright** to **Light**. I think _Dark_ and _Light_ are the most common words in this case.
```suggestion
          <div class="btn-toggle"><i id="symb" class="fa-solid fa-moon fb"><div class="mode-tooltip">Dark/Light Mode</div></i>
```

Reply:
ok